### ISSUE TITLE
What do you mean form  "Softmax is internally computed."

### ISSUE THREAD

**Issue by ErmiaAzarkhalili** [yunjey/pytorch-tutorial#17]
Hi,
Thank you for your great tutorial.

Could you please let me know what you meant from "# Softmax is internally computed." in the [this file](https://github.com/yunjey/pytorch-tutorial/blob/master/tutorials/02%20-%20Logistic%20Regression/main.py#L35-L42)?

**Comment by yunjey** [comment]
You can see [here](http://pytorch.org/docs/nn.html#crossentropyloss) to understand nn.CrossEntropyLoss.

**Comment by ehsanmok** [comment]
Another (more intuitive) option is to return `F.log_softmax(self.linear(inputs))` and use `criterion=nn.NLLLoss()`.
